compass/landice/tests/ismip6_forcing/atmosphere/__init__.py

```diff
@@ -0,0 +1,42 @@
+from compass.testcase import TestCase
+from compass.landice.tests.ismip6_forcing.atmosphere.process_smb \
+    import ProcessSMB
+from compass.landice.tests.ismip6_forcing.atmosphere.process_smb_racmo \
+    import ProcessSmbRacmo
+from compass.landice.tests.ismip6_forcing.configure import configure as \
+    configure_testgroup
+
+
+class Atmosphere(TestCase):
+    """
+    A test case for processing the ISMIP6 atmosphere forcing data.
+    The test case builds a mapping file for interpolation between the
+    ISMIP6 8km polarstereo grid and MALI mesh, regrids the forcing data
+    and rename the ISMIP6 variables to corresponding MALI variables.
+    """
+
+    def __init__(self, test_group):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : compass.landice.tests.ismip6_forcing.Ismip6Forcing
+            The test group that this test case belongs to
+        """
+        name = 'atmosphere'
+        subdir = name
+        super().__init__(test_group=test_group, name=name, subdir=subdir)
+
+        step = ProcessSMB(test_case=self)
```

Comment:
Oh, now I see that this is because the code uses the config setting `base_path_racmo`, but the config file calls it `path_smb_racmo`. I'll note that on the config file.

Reply:
I'm sorry about the hassle!! I have two PRs going on (this one and the original one [https://github.com/MPAS-Dev/compass/pull/377]), and the config file was only updated in the other one. My bad on missing to check the consistency.